DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Imports:
     pacta.portfolio.import, 
     pacta.portfolio.report, 
     pacta.portfolio.utils, 
+    pacta.workflow.utils,
```

Comment:
This dependency doesn't have to do with this repo. 

Since it is introduced in https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/pull/105
I think that package should be installed directly in the `.yml` file there.

Reply:
hmm. but in order to make that whole process work, the package needs to be in the docker image (so it can be called) 🤔

Reply:
Hmmm then lets think about that a bit more carefully. 
Smells a bit fishy to be introducing a dep here that is only relevant to a completely separate repo/ workflow